dm/dm/proto/dmworker.proto

```diff
@@ -383,4 +383,10 @@ message GetWorkerCfgRequest {
 
 message GetWorkerCfgResponse {
     string cfg = 1;
-}
\ No newline at end of file
+}
+
+enum ValidatorOp {
+    InvalidValidatorOp = 0;
```

Comment:
beacuse we want to use openapi reimpl dmctl , how about not add new message to this pb? It's too painful to maintain two copies of the same thing at the same time 😷

Reply:
when will `openapi reimpl dmctl` GA? 

there're a lot of dmctl command related to validation in later pr, do you mean new commands should be written with `openapi`?

Reply:
plan to GA in the next version v6.0, I hope that the new ctl commands will be implemented via openapi if possible, so that there will be less duplication of work later. but this is just weak a suggestion

Reply:
cannot find any commands using openapi, maybe we can do it later, together with other commands.